app/models/rent.rb

```diff
@@ -0,0 +1,4 @@
+class Rent < ApplicationRecord
+  belongs_to :user
```

Comment:
We should add the presence validation of `from` and `to`. 
Also add them to the database.